cross/arm64cl.txt

```diff
@@ -0,0 +1,14 @@
+[binaries]
+c = 'cl'
+cpp = 'cl'
+ar = 'lib'
+windres = 'rc'
+
+[properties]
+c_args = ['-DWINAPI_FAMILY=WINAPI_FAMILY_APP']
```

Comment:
We should add `WindowsApp.lib` in c_link_args, that gives us the actual configuration people cross-compile to ARM64 with. Also needs cpp_args and cpp_link_args?

Reply:
These options aren't mandatory when targeting ARM64 windows - you can build full traditional win32 apps for ARM64, you don't need to target UWP.

And since some time ago, you don't need either `-DWINAPI_FAMILY=WINAPI_FAMILY_APP` (or `-D_ARM_WINAPI_PARTITION_DESKTOP_SDK_AVAILABLE=1`) when targeting ARM32 either, you can build totally regular desktop apps for that nowadays as well. (Since WinSDK 10.0.17763.0, the default define `_ARM_WINAPI_PARTITION_DESKTOP_SDK_AVAILABLE` in ucrt/corecrt.h has been changed from 0 to 1.)

Reply:
Fair enough - just wanted to be sure there's no misunderstanding that it's required for this target.